README.md

```diff
@@ -485,7 +485,7 @@ $ lerna run --scope toolbar-* test
 
 #### --ignore [glob]
 
-Excludes a subset of packages when running a command.
+Excludes a subset of packages when running the `bootstrap` command.
```

Comment:
It's currently `bootstrap` only based on this conversation: https://github.com/lerna/lerna/issues/117#issuecomment-218952045

I think everyone is open to making it available for more commands, we just haven't discussed which commands would use it.